docs/topics/clusterdefinitions.md

```diff
@@ -75,21 +75,20 @@ $ aks-engine get-versions
 | kubernetes-dashboard                                                  | true                | 1                   | Delivers the Kubernetes dashboard component. See https://github.com/kubernetes/dashboard for more info                                                              |
 | rescheduler                                                           | false               | 1                   | Delivers the Kubernetes rescheduler component                                                                                                                       |
-| [cluster-autoscaler](../../examples/addons/cluster-autoscaler/README.md) | false               | 1                   | Delivers the Kubernetes cluster autoscaler component. See https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/azure for more info |
+| [cluster-autoscaler](../../examples/addons/cluster-autoscaler/README.md) | false               | 1                   | Delivers the Kubernetes cluster autoscaler component. See https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/azure for more info; only supported for VMSS clusters on the first agent pool. |
 | [nvidia-device-plugin](../../examples/addons/nvidia-device-plugin/README.md) | true if using a Kubernetes cluster (v1.10+) with an N-series agent pool               | 1                   | Delivers the Kubernetes NVIDIA device plugin component. See https://github.com/NVIDIA/k8s-device-plugin for more info |
 | container-monitoring                       | false               | 1                   | Delivers the Kubernetes container monitoring component |
 | [blobfuse-flexvolume](https://github.com/Azure/kubernetes-volume-drivers/tree/master/flexvolume/blobfuse)                        | true               | as many as linux agent nodes                   | Access virtual filesystem backed by the Azure Blob storage |
 | [smb-flexvolume](https://github.com/Azure/kubernetes-volume-drivers/tree/master/flexvolume/smb)                        | false               | as many as linux agent nodes                   | Access SMB server by using CIFS/SMB protocol |
 | [keyvault-flexvolume](../../examples/addons/keyvault-flexvolume/README.md)                        | true               | as many as linux agent nodes                   | Access secrets, keys, and certs in Azure Key Vault from pods |
 | [aad-pod-identity](../../examples/addons/aad-pod-identity/README.md)                        | false               | 1 + 1 on each linux agent nodes | Assign Azure Active Directory Identities to Kubernetes applications |
-| [pod-security-policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/)                        | false               | 0 | Pod Security Policies enable fine-grained authorization of pod creation and updates |
```

Comment:
what happened to this one?

Reply:
Pod Security Policy is a Frankenstein that needs some distinct TLC. It's a combination of the `KubernetesConfig` property `PodSecurityPolicyConfig` and an addon, and it's stored in the `parts/k8s/addons/` directory (not `containeraddons`).

So, the above documentation is actually wrong as far as I can tell.